docs/capabilities/algokit-composer.md

```diff
@@ -0,0 +1,3 @@
+# AlgoKit composer
+
+TODO
```

Comment:
This can be tackled in future pull requests, it makes sense to have a transaction focussed one that also takes care of adding this.